packages/smooth_app/lib/database/dao_product.dart

```diff
@@ -60,6 +61,26 @@ class DaoProduct extends AbstractSqlDao implements BulkDeletable {
     return result;
   }
 
+  // Returns the no of rows deleted/effected from the table
+  Future<int> clearAll() async {
+    final int rowsDeleted = await localDatabase.database.delete(TABLE_PRODUCT);
+    return rowsDeleted;
+  }
+
+  //Returns the approximate size of the database in MB
+  Future<double> getSize() async {
```

Comment:
Actually it has not much to do with `DaoProduct` (of course I know, for the moment it is the only table); it should be in `LocalDatabase`.